### DISCUSSION TITLE
[Request] Adblock ID for blocking adult sites

### DISCUSSION THREAD

**Discussion by richie3011** [realodix/AdBlockID#1560] [Ideas]
Halo, author AdblockID saya ingin menyarankan untuk membuat satu filter khusus untuk memblokir situs situs dewasa khususnya situs dewasa Indonesia. Filter pemblokiran situs dewasa sangat bermanfaat khususnya bagi orang tua yang khawatir anaknya mengakses situs dewasa. Sebagian situs dewasa bisa di blokir Clean browsing DNS, tapi sebagian lagi seperti situs dewasa Indonesia tidak terjangkau radar Clean browsing DNS & Kemkominfo

### Usulan Sintaks
Untuk contoh Sintaks menggunakan ^$doc atau ^$all. Tapi saya lebih merekomendasikan ^$doc supaya support Adblock lainnya seperti Brave Adblock

**Comment by realodix** [comment]
Halo, mari kita bahas bersama.

Sebelum syntax, ada baiknya kita membahas platform yang ingin dituju, karena syntax akan mengikutinya.

Saya lihat Anda mengincar hanya pada pasar ad blocker, mengapa Anda teratarik untuk memblokirnya hanya pada level ad blocker dan bukan pada level dns / host? Bukannya ad blocker sangat mudah dimatikan oleh user ya?

**Reply by richie3011** [reply]
Saya jarang melihat ada anak anak yg paham tentang adblock/mereka tau cara menonaktifkan adblock untuk membuka situs dewasa 😅. Mungkin anda punya usulan lain yg bisa memblokir situs dewasa 

(Maaf karena saya masih newbie, saya hanya menggunakan syntax ^$doc untuk memblokir situs dewasa di custom rule. Saya berharap kedepannya ada filter khusus untuk memblokir situs-situs porno)

**Reply by realodix** [reply]
Btw pakai ad blocker apa yg anda gunakan untuk memblokir halaman? Saya belum pernah melakukan pemblokiran halaman dengan ad blocker, hanya sekedar menyaring iklan saja.

Saya belum bisa menentukan syntax apa, karena melihat platformnya dulu yang mau dicapai platform apa

**Reply by richie3011** [reply]
Hanya uBlock origin. Saya menggunakan filter dari Blocklists Project untuk memblokir situs situs dewasa

**Reply by 3xploiton3** [reply]
@ChowMein47 bisa pakai filter ini https://nsfw.oisd.nl/
dan jika ada temuan yg belum terblokir silahkan dilaporkan https://oisd.nl/nsfw_reporter

maintaner sangat aktif, 
kekurangan file lumayan besar 9MB+ ada kemungkinan memperlambat kinerja
disarankan pakai blocking dns level atau via proxy

**Comment by realodix** [comment]
Halo, 

Terima kasih atas permintaan fitur filter blocklist. Ini adalah fitur yang sangat bermanfaat bagi masyarakat luas dan harus segera dibuat.

Sayangnya Saya harus menunda fitur tersebut untuk saat ini. Untuk menjaga kemampuan dalam memelihara kualitas AdBlockID secara memadai, Saya harus sangat berhati-hati dalam menyertakan filter.

Jika memungkinkan, harap pertimbangkan untuk menjadi maintainer aktif khusus filter blocklist, dengan cara ini maka filter blocklist bisa segera dibuat dan masyarakat luas bisa segera menikmati manfaatnya.
